django_material_demo/polls/admin_views.py

```diff
@@ -378,13 +413,36 @@ def get_form_class(self):
         return QuestionForm
 
 
+class QuestionFilterForm(forms.Form):
+    layout = Layout('search',
+                    'question_text',
+                    'show_vote')
+
+
+class QuestionFilter(SearchFilterMixin, FilterSet):
+    search_fields = ['question_text', 'creator__account__username',
+                     'choice__choice_text']
+    search = CharFilter(method='keyword_search', label='Search')
```

Comment:
Looks like this line
```
search = CharFilter(method='keyword_search', label='Search')
```
Should be contained in `SearchFilterMixin`, because having this line without the mixin will not work.

Reply:
I tried that before, but the `FilterSet` cannot recognize `search` as one of its filter when it is in the mixin. I will look further into this tomorrow